corehq/apps/analytics/utils/partner_analytics.py

```diff
@@ -177,3 +174,12 @@ def send_partner_emails(year, month):
             text_content=render_to_string("analytics/email/partner_analytics_report.txt", context),
             file_attachments=[{'file_obj': file_obj, 'title': filename, 'mimetype': 'text/csv'}],
         )
+
+
+def _get_csv_value(value):
+    if isinstance(value, str):
+        return value
+    try:
+        return int(value)
+    except ValueError:
+        return bytes(value)
```

Comment:
I find it surprising that we ever want to write raw bytes to a CSV row. Is this branch ever used?